apps/crm/README.md

```diff
@@ -1,7 +1,146 @@
+# Saleor App CRM
+
 ![Hero image](https://user-images.githubusercontent.com/249912/71523206-4e45f800-28c8-11ea-84ba-345a9bfc998a.png)
 
-<div align="center">
-  <h1>Saleor App CRM</h1>
-</div>
 
-Visit [CRM App documentation](https://docs.saleor.io/docs/3.x/developer/app-store/apps/crm) to learn how to configure and develop the app locally.
+_CRM_ is a Saleor App that allows providing customer data to external services. Here are the currently available providers:
+
+- [Mailchimp](http://mailchimp.com/)
+- [Twilio Segment](https://segment.com/) [_coming soon_]
+- [RudderStack](https://www.rudderstack.com/) [_coming soon_]
+
+There are two main features CRM App provides:
+
+- Initial indexing of existing Saleor customers and sending them to the provider contact list
+- Synchronizing new customers between Saleor and the provider
+
+:::caution
+
+To successfully configure the CRM App, you must:
+
+- Have a Mailchimp account
+- Have a Saleor user with `MANAGE_APPS` permission
+
+:::
+
+## Installation
+
+The App can be installed from [Saleor App Store](developer/app-store/overview.mdx). To install the App, you must have `MANAGE_USERS` and `MANAGE_APPS` [permissions](developer/permissions.mdx#app-permissions).
```

Comment:
The links won't work in the repository. Do we have a plan for addressing that?

Reply:
These must be absolute links